html/pages/apps.inc.php

```diff
@@ -1,4 +1,26 @@
 <?php
+/*
```

Comment:
I've been thinking about this copyright you've added to files you've changed. I'm not 100% sure on the position but I would say that we shouldn't be adding a copyright that seems to claim it's done by LibreNMS contributors. Old historic files weren't written by us so I think you should remove this / these.